quantifier/src/main/java/edu/illinois/cs/cogcomp/quant/lbj/QuantitiesClassifier$$1.java

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
I think these outputs are slightly modified after being generated from lbjava. But we can definitely change it so that it generates it on the fly, but we have to make sure about its functionality. How about checking this in like this, and delegating this to another issue?